docs/sql-guide/statements/statement-insert.md

```diff
@@ -78,12 +78,9 @@ INSERT INTO services (_id, servicelist, price) VALUES (1, 'free deliveries on or
 INSERT INTO services (_id, servicelist, price) VALUES (2, 'local postage per item', 2.20);
 ```
 
-### INSERT values into single column identified by `_id`
-
+### INSERT a value with an associated unixtimestamp for a [IDSETQ() data type](/docs/sql-guide/data-types/data-type-idsetq)
```

Comment:
A heading introduces something, you don't put hyperlinks into it.
If you want a hyperlink it would go into text beneath

For example:
```
### INSERT for IDSETQ data type

The following statement inserts a value with an associated `unixtimestamp` into a <row/column/record???> with an IDSETQ() data type.

STATEMENT GOES HERE

```

Then at the end of the page you can do the following:

```
## Further information

* [IDSETQ() data type](/docs/sql-guide/data-types/data-type-idsetq)